mod-pubsub-server/src/main/java/org/folio/services/ConsumerService.java

```diff
@@ -15,7 +15,7 @@
    *
    * @param eventTypes list of event types that specified module is subscribing to receive
    * @param params     Okapi connection params
-   * @return future with true if succeeded
+   * @return a future completed with a result
```

Comment:
Should we make this description more direct, like in https://github.com/folio-org/mod-pubsub/pull/200/files this PR?

Reply:
fixed